src/components/Question.js

```diff
@@ -40,6 +43,10 @@ class Question extends Component {
     this.setState({ userAnswerIndex: event.target.value });
   }
 
+  handleCorrectAnswer(answer) {
```

Comment:
I think all of this logic could be handled in `handleSubmit`

Reply:
I agree, esp since its not being called by the nav button anymore.